Week-2/q1Article/article.css

```diff
@@ -0,0 +1,96 @@
+
+.article1{
+  text-align: center;
+}
+
+/*.column {
```

Comment:
commented piece of code should not be merged

Reply:
ok i will remove the commented code